crates/interpreter/src/instruction_result.rs

```diff
@@ -237,20 +257,3 @@ impl From<InstructionResult> for SuccessOrHalt {
         }
     }
 }
-
-#[macro_export]
-macro_rules! return_ok {
-    () => {
-        InstructionResult::Continue
-            | InstructionResult::Stop
-            | InstructionResult::Return
-            | InstructionResult::SelfDestruct
-    };
-}
-
-#[macro_export]
-macro_rules! return_revert {
-    () => {
-        InstructionResult::Revert | InstructionResult::CallTooDeep | InstructionResult::OutOfFund
-    };
-}
```

Comment:
Would be good to add a test that covers all cases.

Something likes
```
let temp = InstructionResult::Continue;

match temp {
   return_ok!() => (),
   return_revert!() => (),
   return_error!() => (),
}
```

Reply:
Done

Reply:
It's only after your [add exhaustive check for return_ macros](https://github.com/bluealloy/revm/pull/956/commits/95f7e9d409329748caa222797388974004ccd2ce) commit that I've understood what exactly you wanted to be tested about the newly-added macros. 😬
My bad!

Also, thanks for fixing the name of the test function I've introduced! 🙌